src/app/components/navbar/navbar.component.scss

```diff
@@ -0,0 +1,33 @@
+.sidenav-container {
+  height: 100%;
+}
+
+.sidenav {
+  width: 200px;
+}
+
+.sidenav .mat-toolbar {
```

Comment:
If you're using `sass`, you should place all child styles under parent selector

```
.sidenav {
    .mat-toolbar {
         ...
     }
}
```

Reply:
css thing from generating navbar as a whole component